setup.py

```diff
@@ -111,12 +58,14 @@ def pip(filename):
     version=__import__('udata').__version__,
     description=__import__('udata').__description__,
     long_description=long_description,
+    long_description_content_type='text/markdown',
     url='https://github.com/opendatateam/udata',
     author='Opendata Team',
     author_email='opendatateam@data.gouv.fr',
     packages=find_packages(),
     include_package_data=True,
     install_requires=install_requires,
+    setup_requires=['setuptools>=38.6.0'],
```

Comment:
👍 Do this in `udata-gouvfr`?

Reply:
Yes, it's done. We need to do this for all packages